tests/test_resources/test_invalid_namespace.yml

```diff
@@ -0,0 +1,12 @@
+name: 'dockertest'
+description: 'This is a test container. For testing the MPL pipelines, not to be deployed anywhere.'
+stages:
+  build: Docker Build
+  test: Docker Test
+  deploy: Kubernetes Deploy
+maintainer: [ 'Marketplace' ]
+deployment:
+  properties:
+    env:
+      - key: WITH_NAMESPACE
+        all: "prometheus-gateway.{namespace}.svc.cluster.local"
```

Comment:
In accordance with https://vandebron.atlassian.net/browse/TECH-302, I think let's get rid of vdb specific URLs and names in this PR as well

Reply:
I agree, but isn't this a pretty neutral/common/normal url?

Reply:
Yeah okay fair enough. The test_project.yml will be modified I guess in the other card, so then nothing to do here